scripts/travis/android/linux-android.sh

```diff
@@ -0,0 +1,54 @@
+#! /bin/bash
+set -euxo pipefail
+
+[ "$ABI" = "armv7a" ] && host=$ABI-linux-androideabi || host=$ABI-linux-android
+sysroot="$ANDROID_NDK_ROOT/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/lib/$host"
+
+case "$ABI" in
+    "x86_64")
+        arch=$ABI
+        ;;
+    "i686")
+        arch=x86
+        ;;
+    "aarch64")
+        arch=arm64-v8a
+        ;;
+    "armv7a")
+        arch=armeabi-v7a
+        sysroot="$ANDROID_NDK_ROOT/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/lib/arm-linux-androideabi"
+        ;;
+    *) echo "unrecognized android ABI."; exit 1;;
+esac
```

Comment:
For readability, I'd suggest to rewrite this a bit more explicitly:

```
case "$ABI" in
    "x86_64")
        arch=...
        host=...
        sysroot=...
        ;;
    "i686")
        arch=...
        host=...
        sysroot=...
        ;;
    ....
esac
```

In other words, duplicate host in sysroot in every branch. The duplication is small, and it will be immediately clear how each abi is mapped to these variables.

Reply:
Sure :+1: